app/styles/ember-appuniversum/_u-flex.scss

```diff
@@ -105,6 +105,29 @@ $au-flex-utilities-responsive: false !default;
   align-items: baseline !important;
 }
 
+// Child elements alignment
+
+.au-u-flex__self-start {
```

Comment:
Another nitpick, but it feels strange to me to use BEM-like naming for util classes 😄. We're already using modifiers for the other things so it probably makes sense to use BEM here as well, but I just wondered if something like `au-u-self-start` / `au-u-flex-self-start` wouldn't be "better"?